dm/syncer/syncer.go

```diff
@@ -931,10 +940,15 @@ func (s *Syncer) addJob(job *job) error {
 	case flush:
 		s.addCount(false, adminQueueName, job.tp, 1, job.targetTable)
 		s.jobWg.Add(1)
+		job.wg.Add(s.cfg.WorkerCount)
 		s.dmlJobCh <- job
-		s.jobWg.Wait()
```

Comment:
if we don't wait s.jobWg, seems no need to `s.jobWg.Add(1)` in L942?

Reply:
Other jobs like DDL still need to wait all previous done.